REVIEWING.md

```diff
@@ -0,0 +1,13 @@
+# Reviewing guide
+
+## Values and guidelines
+
+All reviewers must abide by the [Code of Conduct](CODE_OF_CONDUCT.md); they are also protected by the Code of Conduct.
+A reviewer should not tolerate poor behavior and is encouraged to [report any behavior](CODE_OF_CONDUCT.md#Reporting_violations) that violates the Code of Conduct.
+
+Everyone participating must follow our [Community Participation Guidelines](CODE_OF_CONDUCT.md#Community_Participation_Guidelines)
```

Comment:
There seems to be a repeated link to the same file/section - code of conduct. I've opened https://github.com/mdn/project-template/pull/20 to correct this.

Reply:
All other files and changes look good. Not merging it just yet in case https://github.com/mdn/project-template/pull/20 changes need to be replicated.

Reply:
Good catch, fixed in https://github.com/mdn/express-locallibrary-tutorial/pull/139/commits/23bf70a84c1ec541045f0f3b280f58f905e5665a